src/style.css

```diff
@@ -0,0 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+:root {
+  ---accent-page-color: #00acdc;
+}
+
+/* #region reset */
+body,
+p,
+h1 {
+  margin: 0;
+}
+/* #endregion */
+
+.product-card {
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+
+  width: 200px;
+}
+
+.product-card__image {
+  margin-top: 16px;
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__name {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  margin-top: 4px;
+  align-self: flex-start;
+
+  color: #616070;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card-rate {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: 16px;
+}
+
+/* #region stars */
```

Comment:
here I suppose we need separate stars styles to another file and add these styles to the style.css via @import